build_projects/update-dependencies/update-dependencies.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>Updates the repos dependencies</Description>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RestoreSources>https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;https://api.nuget.org/v3/index.json</RestoreSources>
```

Comment:
What do you think about formatting this so it is more readable?

For example, check out how this is formatted:

https://github.com/dotnet/core-setup/blob/2a19751530a6abcad593883c4381a6b4893f809e/dir.props#L46-L55